### PR TITLE
326: Loading Indicator

### PR DESCRIPTION
**Pull request by pcberg**
* Question: Do we like the component name "LoadingIndicator"?
* I added the Button loading indicator sample to both Button.md and LoadingIndicator.md. The other changes in Button.md is cleanup, prettify seemed to have added some end of line weirdness. Hopefully this sticks

### CODE REVIEW

File: src/components/Button/Button.jsx [hunk 1]
```diff
@@ -45,15 +51,20 @@ const Button = ({
...
       className
     )}
     onClick={onClick}
     ref={innerRef}
+    disabled={loading}
+    aria-busy={loading}
     {...attrs}
   >
-    {children}
+    <span className="rvt-button__content" role="buttonContent">
```
Comment (ri-pandey) [inline src/components/Button/Button.jsx:64]: You could consider wrapping the button text in this span only if the loading prop is true, to mimic Rivet more closely.

Comment (pcberg) [reply src/components/Button/Button.jsx]: Good idea, thanks Rishi! I improved it.

**Review by ri-pandey — APPROVED** [review]

**Comment by pcberg** [comment]
I merged this, as always I'm happy to revisit should anybody notice something after the fact. I realize my turn around on this PR was fast. Thanks!

**Pull request merged**